### PR TITLE
Change from parsing the issue body to taking email as input from another action

### LINKED ISSUE #20: [FEATURE] - Refactor to take input from the actions-parse-issue action
Currently the action parses the issue body for the email, rip this out and take input from previous action

### PR DESCRIPTION
**Pull request by Chocrates**
Resolves #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,19 +112,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Get issue data
+      - name: Get User Input
+        id: get_input
+        uses: jasonmacgowan/actions-parse-issue@master
+        env:
+          ADMIN_TOKEN: ${{ secrets.ADMTIN_TOKEN }}
```
Comment (jasonmacgowan) [inline README.md]: we don't need the admin token

File: README.md [hunk 2]
```diff
@@ -119,19 +112,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Get issue data
+      - name: Get User Input
+        id: get_input
+        uses: jasonmacgowan/actions-parse-issue@master
```
Comment (jasonmacgowan) [inline README.md]: needs to change to `jasonmacgowan/parse-issue@master` as of this morning

File: README.md [hunk 3]
```diff
@@ -119,19 +112,25 @@ jobs:
...
+          EMAIL: ${{ steps.get_input.outputs.['Email of Requester']}}
         env:
           ADMIN_TOKEN: ${{secrets.ADMIN_TOKEN}}
       - name: Comment on Issue
         uses: froi/add-comment-action@v1
         with:
-          message: {{ steps.get-issue-data.message }}
-          status: {{ steps.get-issue-data.stepStatus }}
+          message: { { steps.get-issue-data.message } }
+          status: { { steps.get-issue-data.stepStatus } }
```
Comment (jasonmacgowan) [inline README.md]: I think these template calls need to change to `${{ ... }}`

**Comment by Chocrates** [comment]
@jasonmacgowan good feedback.  I have pushed updates for all of them and altered the sample workflow a bit to use your new custom extractor.

File: README.md [hunk 4]
```diff
@@ -119,19 +112,25 @@ jobs:
...
     steps:
       - uses: actions/checkout@v1
-      - name: Get issue data
+      - name: Get User Input
+        id: get_input
+        uses: jasonmacgowan/parse-issue@master
+        with:
+          extract_email: '<p>Email of Requester:\s*(.*)</p>'
+        env:
+          ADMIN_TOKEN: ${{ secrets.ADMTIN_TOKEN }}
```
Comment (jasonmacgowan) [inline README.md]: Let's remove this or fix secrets reference to ADMIN_TOKEN

Comment (Chocrates) [reply README.md]: apologies, don't know why I marked it as resolved if I didn't do anything about it.  
I will get it removed today.

**Pull request merged**